api/api.go

```diff
@@ -320,6 +388,16 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 	respond(w, res)
 }
 
+func regexpAny(re *regexp.Regexp, ss []string) bool {
```

Comment:
I'm assuming `continue` in the route config is used somewhat infrequently, so this function will probably only have a single element in `ss` for the majority of cases.